python/paddle/distributed/auto_parallel/static/completion.py

```diff
@@ -1198,6 +1313,9 @@ def _complete_high_order_grad_annotation(self, serial_main_program=None):
             This function is temporary to support high order gradient, and will be removed in the future.
         """
 
+        if serial_main_program._appending_grad_times < 2:
+            return
```

Comment:
why ?

Reply:
This function is for the high-order grad model, so the normal model doesn't need to execute.